src/components/Editor/Breakpoints.js

```diff
@@ -61,11 +59,8 @@ class Breakpoints extends Component<Props> {
   }
 }
 
-export default connect(
-  state => ({
-    breakpoints: getVisibleBreakpoints(state),
-    selectedSource: getSelectedSource(state),
-    sourceMetaData: getSourceMetaData(state, getSelectedSource(state).id)
-  }),
-  dispatch => bindActionCreators(actions, dispatch)
-)(Breakpoints);
+export default connect(state => ({
```

Comment:
Hmm I don't know if we want to remove the actions entirely here. If BreakPoints uses any we should import them here and put them in an object as the second argument to `connect()`

Reply:
@wldcordeiro can you give an example?

Reply:
@wldcordeiro Remove because Breakpoints.js part uses no actions and does not pass other actions as props to Breakpoint.js.

Reply:
I'd like to support @Fischer-L in this. 
I'd like to see us explicitly importing and connection actions. If there is no action used, make that clear.